components/fxa-client/android/src/main/java/mozilla/appservices/fxaclient/FirefoxAccount.kt

```diff
@@ -187,6 +187,34 @@ class FirefoxAccount(handle: FxaHandle, persistCallback: PersistCallback?) : Aut
         }.getAndConsumeRustString()
     }
 
+    /**
+     * Fetches the user's manage-account url.
+     *
+     * This performs network requests, and should not be used on the main thread.
```

Comment:
I had to mark these as potentially performing network requests, because the only way to get at `uid` and `email` currently is to fetch the profile info, which will trigger a network request if it's not cached locally.  That seems suboptimal, and similar to the issues @grigoryk was hitting over in https://github.com/mozilla/application-services/issues/483.